### DISCUSSION TITLE
Specify image via http API request

### DISCUSSION THREAD

**Discussion by Radins** [containrrr/watchtower#1449] [Q&A] [answered]
How  do I specify the images to be updated via http request to the API?

Looking into de the documentation, I couldn't find anything, but in the source code, the function `func (handle *Handler) Handle(w http.ResponseWriter, r *http.Request)` seems to check if any images was passed through query parameters, and only update those images. But testing it locally, every request to `/v1/update` update all containers.

Did I miss something while reading the code, or I'm doing something wrong?

**Comment by piksel** [comment, marked as answer]
This feature is not released yet, but you can try it using the `containrrr/watchtower:latest-dev` image.
